src/references/auth.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Authentication
+---
+
+## Authentication Sequence
+
+1. The User opens up a browser to check his cozy, using his *cozy app domain* (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...)
+2. he's redirected to his *cozy user domain* (`user.mycozy.cloud`)
+3. The browser request the *cozy user domain*, which send back the authentication form. The user fills his password, sending it back to the  *cozy user domain*
+4. The server *cozy user domain* set an auth cookie for *cozy user domain* ONLY
+5. A redirect status response code (302) for the requested app domain (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...) is sent to the browser
+5. The browser request the *cozy app domain* with a [jwt code](https://jwt.io/)
```

Comment:
```suggestion
5. The browser requests the *cozy app domain* with a [jwt code](https://jwt.io/)
```

Reply:
Is this using the `Authorization` header here?

Reply:
Nope. The 302 redirection from user.mycozy.cloud to user-home.mycozy.cloud adds a `?code=xxxxx` to the URL where xxxx is the JWT.

Ok, it's maybe usefull to write it explicitely

Reply:
I think so, it's more clear for me now, thanks 👍